crates/solver-service/src/apis/quote/generation.rs

```diff
@@ -636,7 +642,7 @@ impl QuoteGenerator {
 					"asset": output.asset.to_string(),
 					"amount": output.amount.clone(),
 					"receiver": output.receiver.to_string(),
-					"oracle": format!("0x{:040x}", alloy_primitives::Address::from_slice(&selected_oracle.0)),
+					"oracle": format!("0x{:040x}", alloy_primitives::Address::from_slice(&output_oracle.0)),
```

Comment:
We were wrongfully setting oracle here to input oracle instead of output oracle. cc @nahimterrazas @NicoMolinaOZ